core/src/test/java/nl/nn/adapterframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -205,4 +205,32 @@ public void testAddMonth() throws Exception {
 		Message result=sendMessage("dummy");
 		assertEquals("<result><rowsupdated>1</rowsupdated></result>", result.asString());
 	}
+	
+	public void testOutputFormat(DocumentFormat outputFormat, String expected) throws Exception {
+		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR, tINT) VALUES ('1', 2, 3)");
+		sender.setColumnsReturned("tKEY, tVARCHAR, tINT");
+
+		sender.setOutputFormat(outputFormat);
+		sender.configure();
+		sender.open();
+
+		Message result = sendMessage("dummy");
+		assertEquals(expected, result.asString());
+	}
+
+	@Test
+	public void testOutputFormatDefault() throws Exception {
+		testOutputFormat(null,"<result><rowset><row number=\"0\"><field name=\"TKEY\">1</field><field name=\"TVARCHAR\">2</field><field name=\"TINT\">3</field></row></rowset></result>");
+	}
+
+	@Test
+	public void testOutputFormatXml() throws Exception {
+		testOutputFormat(DocumentFormat.XML, "<result><fields><name>TKEY</name><name>TVARCHAR</name><name>TINT</name></fields><rowset><row><field>1</field><field>2</field><field>3</field></row></rowset></result>");
```

Comment:
volgens mij staat er ergens een if verkeerd om? :)

Reply:
want?

Reply:
Ik zie de fielddefinition niet terug? Die staat toch standaard aan?

Reply:
In FixedQuerySenderTest staat `sender.setIncludeFieldDefinition(false);`, om er geen last van te hebben.
Ik zal op een andere plek testen maken inclusief veld definities.